wazo_agid/modules/call_recording.py

```diff
@@ -9,24 +9,34 @@
 
 
 def call_recording(agi, cursor, args):
+    calld = agi.config['calld']['client']
+    channel_id = agi.env['agi_uniqueid']
     if agi.get_variable('WAZO_CALL_RECORD_ACTIVE') == '1':
-        _disable_call_recording(agi, cursor, args)
+        _disable_call_recording(agi, calld, channel_id)
     else:
-        _enable_call_recording(agi, cursor, args)
+        _enable_call_recording(agi, calld, channel_id)
 
 
-def _enable_call_recording(agi, cursor, args):
+def _enable_call_recording(agi, calld, channel_id):
     record_filename = agi.get_variable('XIVO_CALLRECORDFILE')
```

Comment:
Would it be bad to remove the `XIVO_CALLRECORDFILE` check before calling the API?

Reply:
I don't know, it was the previous logic. Since the API won't handled it, it would leave it there